internal/no-std-stub/src/lib.rs

```diff
@@ -0,0 +1,119 @@
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
Instead of implementing {bcmp, memcmp, memcpy, memmove, memset} could we depend on [compiler_builtins](https://crates.io/crates/compiler_builtins) w/ the [`mem`](https://docs.rs/compiler_builtins/0.1.36/compiler_builtins/mem/index.html) feature as a dependency? Since it provides them, the linker should be able to do the right thing.

Reply:
done

Reply:
@connorkuehl @npmccallum: I just saw that `compiler_builtins` needs `nightly` with quite a lot of `features`. When we move to stable we have to revert using it.